google-cloud-tasks/lib/google/cloud/tasks/v2beta2/credentials.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Google LLC
```

Comment:
Now that this file has been moved the old file `lib/google/cloud/tasks/credentials.rb` should be removed.